liquibase-dist/pom.xml

```diff
@@ -1,301 +1,550 @@
-                        <id>make-assembly</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>single</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-
-            <!-- install the shaded jar as liquibase-core like the final artifact ends up as -->
-            <plugin>
-                <artifactId>maven-install-plugin</artifactId>
-                <configuration>
-                    <artifactId>liquibase-core</artifactId>
-                    <file>${project.build.directory}/liquibase-${project.version}.jar</file>
-                    <sources>${project.build.directory}/liquibase-${project.version}-sources.jar</sources>
-                    <pomFile>${project.build.directory}/flattened-pom/release.pom.xml</pomFile>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>custom-install</id>
-                        <phase>install</phase>
-                        <goals>
-                            <goal>install-file</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-
-<!--            <plugin>-->
-<!--                <groupId>org.jacoco</groupId>-->
-<!--                <artifactId>jacoco-maven-plugin</artifactId>-->
-<!--                <executions>-->
-<!--                    <execution>-->
-<!--                        <id>report-aggregate</id>-->
-<!--                        <phase>test</phase>-->
-<!--                        <goals>-->
-<!--                            <goal>report-aggregate</goal>-->
-<!--                        </goals>-->
-<!--                    </execution>-->
-<!--                    <execution>-->
-<!--                        <goals>-->
-<!--                            <goal>prepare-agent</goal>-->
-<!--                        </goals>-->
-<!--                        <phase>none</phase>-->
-<!--                    </execution>-->
-<!--                </executions>-->
-<!--            </plugin>-->
-        </plugins>
-    </build>
-
-    <profiles>
-        <profile>
-            <id>liquibase-pro</id>
-
-            <activation>
-                <file>
-                    <exists>../../liquibase-pro/pom.xml</exists>
-                </file>
-            </activation>
-
-            <dependencies>
-                <dependency>
-                    <groupId>com.datical.lb</groupId>
-                    <artifactId>liquibase-pro</artifactId>
-                    <version>${project.version}</version>
-                </dependency>
-            </dependencies>
-        </profile>
-    </profiles>
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Spoke to @StevenMassaro  about this. Will assign to @wwillard7800 instead for review.

Reply:
I'm fixing up the change to the formatting :)

Reply:
Just to clarify, @ap-liquibase is asking for a review from @wwillard7800 because I am not familiar enough with these maven concepts to provide any feedback, not because I'm crotchety about diff formatting.